Python/HAT/metrics.py

```
@@ -0,0 +1,168 @@
+"""
+Hypergraph metrics:
+
+1. matrix entropy
+2. average distance
+3. clustering coefficient
+4. nonlinear_eigenvector_centrality
+"""
+
+import numpy as np
+import networkx as np
```

Ruff (0.8.2)

Redefinition of unused `np` from line 10

(F811)